app/lib/tariff_synchronizer/base_update.rb

```diff
@@ -155,6 +155,16 @@ def import!
       raise NotImplementedError
     end
 
+    def cache_key_with_version
+      [
+        'tariff-update',
+        filename,
```

Comment:
Probably the filename is enough, here but no harm including the other fields

Reply:
Yeh, it was like that originally but then changed it to follow what ActiveRecord was doing man_shrugging